{{ cookiecutter.project_slug }}/.github/workflows/backend.yml

```diff
@@ -112,7 +112,7 @@ jobs:
           tags: |
             type=ref,event=branch
             type=sha
-            type=raw,value=latest,enable={{ "${{ is_default_branch }}" }}
+            type=raw,value=latest,enable={{ "{{is_default_branch}}" }}
```

Comment:
What is the purpose of `is_default_branch`? Is it supposed to be evaluated when running cookiecutter, or in CI after the cookiecutter has rendered the project?

Reply:
This needs to be documented in its usage. I found nothing.

https://github.com/collective/cookiecutter-plone-starter/search?q=is_default_branch

If the environment variable is specific to GHA, then a reference to its usage needs to be documented.

As far as this change, it sounds like we need the rendered project to end up with this:

```
type=raw,value=latest,enable=$is_default_branch
```

Please verify. If so, then this should yield the desired rendered result:

```
type=raw,value=latest,enable=$is_default_branch
```

Reply:
@stevepiercy: There is a [reference](https://github.com/docker/metadata-action#latest-tag) in the description of this pull request. This is implemented by the `docker/metadata-action`

Reply:
If you expect users to search the PRs or issue trackers for documentation, then you are doing it wrong. The reference must be in the README, at least, as minimal documentation.

For this change, does it render as the following, as specified in the reference?

```
type=raw,value=latest,enable={{is_default_branch}}
```

I think that is the intended end result, correct?